kirin/utils.py

```diff
@@ -333,3 +335,23 @@ def get_lock(logger, lock_name, lock_timeout):
         if locked:
             logger.debug("releasing lock %s", lock_name)
             lock.release()
+
+
+def log_exception(exception, origin):
+    """
+    log all exceptions
+    """
+    logger = logging.getLogger(__name__)
+    message = ""
+    if hasattr(exception, "data"):
+        message = exception.data
+    error = "{ex} {data} {orig}".format(ex=exception.__class__.__name__, data=message, orig=origin)
```

Comment:
How about using separators here? Because we don't know exactly what `exception.data` might look like. Something like that?

```suggestion
    error = "{ex}: {data} - {orig}".format(ex=exception.__class__.__name__, data=message, orig=origin)
```